test/test.c

```diff
@@ -11051,7 +11053,7 @@ _sendRequest(void *closure)
 
     nats_Sleep(250);
 
-    s = natsConnection_RequestString(&msg, arg->nc, "foo", "Help!", 2000);
+    s = natsConnection_RequestString(&msg, arg->nc, "foo", arg->string, 2000);
```

Comment:
Did not have to change. The point of this test is to test parallel calls to Request() APIs. The request test here does not matter.